lib/cockpit-components-listing.css

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
Can you also add a comment about the possible endgame here. For example:

"Goal, is to move have the entire CSS for the listing view outside of the base package."